.github/scripts/build-dasharo-payload-pkg-qemu.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+make -C BaseTools
+source ./edksetup.sh
+
+export EDK2_PLATFORMS_PATH="$WORKSPACE/edk2-platforms"
+export PACKAGES_PATH="$WORKSPACE:\
+$WORKSPACE/ipxe/src/bin-x86_64-efi-sb:\
+$EDK2_PLATFORMS_PATH/Platform/Intel:\
+$EDK2_PLATFORMS_PATH/Silicon/Intel:\
+$EDK2_PLATFORMS_PATH/Features/Intel:\
+$EDK2_PLATFORMS_PATH/Features/Intel/Debugging:\
+$EDK2_PLATFORMS_PATH/Features/Intel/Network:\
+$EDK2_PLATFORMS_PATH/Features/Intel/OutOfBandManagement:\
+$EDK2_PLATFORMS_PATH/Features/Intel/PowerManagement:\
+$EDK2_PLATFORMS_PATH/Features/Intel/SystemInformation:\
+$EDK2_PLATFORMS_PATH/Features/Intel/UserInterface:\
+$EDK2_PLATFORMS_PATH/Drivers"
+
+function build_dasharopayloadpkg_64() {
+    build -a IA32 -a X64 -t GCC5 \
+          -D BOOTLOADER=COREBOOT \
```

Comment:
Here are some other options passed by coreboot when building EDK, I think we need to test the build with them (some can be dropped if this breaks the build and shouldn't actually work in this setup):
```
ABOVE_4G_MEMORY=FALSE
BOOTSPLASH_IMAGE=TRUE
CAPSULE_SUPPORT=TRUE
CAPSULE_MAIN_FW_GUID=32a75a2a-17ff-4d1b-88c2-e9dff5db53e7
CPU_TIMER_LIB_ENABLE=FALSE
DASHARO_SYSTEM_FEATURES_ENABLE=TRUE
DISABLE_SERIAL_TERMINAL=FALSE
NETWORK_IPXE=TRUE
OPAL_PASSWORD_ENABLE=TRUE
PERFORMANCE_MEASUREMENT_ENABLE=TRUE
PRIORITIZE_INTERNAL=TRUE
PS2_KEYBOARD_ENABLE=TRUE
SATA_PASSWORD_ENABLE=TRUE
SECURE_BOOT_DEFAULT_ENABLE=FALSE
SECURE_BOOT_ENABLE=TRUE
SERIAL_TERMINAL=TRUE
SETUP_PASSWORD_ENABLE=TRUE
SIO_BUS_ENABLE=TRUE
UART_ON_SUPERIO=TRUE
USE_CBMEM_FOR_CONSOLE=TRUE
```

Reply:
> all except `CAPSULE_SUPPORT=TRUE` and `NETWORK_IPXE=TRUE`).

They are both kinda important, so should really be present.

For `NETWORK_IPXE=TRUE` I suspect a missing `DasharoPayloadPkg/NetworkDrivers/ipxe.efi`, so CI just needs to run something like `cp ipxe/src/bin-x86_64-efi-sb/ipxe.efi DasharoPayloadPkg/NetworkDrivers/`.

What didn't work about `CAPSULE_SUPPORT=TRUE`?  It builds for me locally on your branch.

Reply:
Build from a clean worktree also passes.  Maybe that failure was a result of multiple builds or lack of `CAPSULE_MAIN_FW_GUID` which is required for `CAPSULE_SUPPORT=TRUE`.  I thought you'd just put all of them in and see if anything fails.